cypress/e2e/awx/resources/credentials.cy.ts

```diff
@@ -493,3 +482,307 @@ describe('Credentials', () => {
+        cy.getByDataCy('create-credential').click();
+        cy.verifyPageTitle('Create Credential');
+        cy.getByDataCy('name').type(credentialName);
+        cy.wait('@getCredentialTypes').then(() => {
+          cy.singleSelectByDataCy('credential_type', `${item.name}`, true);
+          if (Array.isArray(item.required)) {
+            (item.required as { field: string; dataCy: string }[]).forEach((credentialType) => {
+              switch (item.name) {
+                case 'HashiCorp Vault Secret Lookup':
+                  cy.get(`[data-cy="${credentialType.dataCy}"]`).type(`${credentialType.field}`);
+                  cy.selectDropdownOptionByResourceName('api-version', 'v1');
+                  break;
+                case 'Ansible Galaxy/Automation Hub API Token':
+                  cy.singleSelectByDataCy(
+                    'organization',
+                    `${(this.globalOrganization as Organization).name}`
+                  );
+                  cy.get(`[data-cy="${credentialType.dataCy}"]`).type(`${credentialType.field}`);
+                  break;
+                case 'GPG Public Key':
+                  cy.get('#gpg-public-key').type(`${credentialType.field}`);
+                  break;
+                case 'Terraform backend configuration':
+                  cy.get('[data-cy="configuration-form-group"]').type(`${credentialType.field}`);
+                  break;
+                default:
+                  cy.get(`[data-cy="${credentialType.dataCy}"]`).type(`${credentialType.field}`);
+              }
+            });
+          }
+          cy.intercept('POST', awxAPI`/credentials/`).as('created');
+          cy.getByDataCy('Submit').click();
+          cy.wait('@created');
+          cy.verifyPageTitle(credentialName);
+          cy.clickPageAction('delete-credential');
+          cy.get('#confirm').click();
+          cy.clickButton(/^Delete credential/);
+          cy.verifyPageTitle('Credentials');
+        });
+      });
+    });
+  });
+});
+
+describe('Credentials Tabbed View - Job Templates', function () {
+  let machineCredential: Credential;
+  let createdAwxUser: AwxUser;
+  let awxOrganization: Organization;
+  let createdAwxTeam: Team;
+  let awxInventory: Inventory;
+
+  beforeEach(function () {
+    cy.awxLogin();
+    cy.createAwxOrganization().then((awxOrg) => {
+      awxOrganization = awxOrg;
+      cy.createAwxUser(awxOrganization).then((awxUser) => {
+        createdAwxUser = awxUser;
+        cy.createAWXCredential({
+          kind: 'machine',
+          organization: awxOrganization.id,
+          credential_type: 1,
+        }).then((cred) => {
+          machineCredential = cred;
+        });
+      });
+      cy.createAwxTeam(awxOrganization).then((awxTeam) => {
+        createdAwxTeam = awxTeam;
+      });
+      cy.createAwxInventory().then((inv) => {
+        awxInventory = inv;
+      });
+    });
+  });
+
+  after(() => {
```

Comment:
I believe this should be an afterEach block, since these resources are created in a beforeEach block; will help with proper resource cleanup.

Reply:
since there is only one test in the `describe` block, using `after` is fine and works as expected. `afterEach` would be when we have multiple it blocks inside the `describe` and we want the resource cleanup to happen `after` each test